packages/next-app/.env.sample

```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_INFURA_ID=""
+NEXT_PUBLIC_INFURA_ID="7065a36adc4f490e97992b20d22c4626"
```

Comment:
Dont put your infura key in the sample. Create a new file called .env.local and fill out your info there.